config/recipe-wayfire-nvidia-gdm.yml

```diff
@@ -24,3 +24,4 @@ modules:
   - from-file: common-scripts.yml
   - from-file: gdm-scripts.yml
   - type: signing
+  - from-file: common-systemd.yml
```

Comment:
The -gdm images don't have sddm, they have gdm, so this shouldn't be included for those